spec/controllers/api/v1/project_contents_controller_spec.rb

```diff
@@ -24,6 +17,14 @@
   let(:primary_content) { project.primary_content }
 
   describe "#index" do
+    let!(:acl) do
```

Comment:
They could be. I could have them at the top level since it causes a validation error in the #destroy tests.